cmd/prcompFormat/prcompFormat.go

```diff
@@ -42,9 +41,11 @@ func prcompFormat(infile string, outfile string) {
 		allValid = true
 		if !(currentBase == dna.N || currentBase == dna.Gap) {
 			for j := 1; j < len(records); j++ {
-				//fmt.Printf("i = %d, j = %d\n", i, j)
 				if records[j].Seq[i] != currentBase && records[j].Seq[i] != dna.Gap && records[j].Seq[i] != dna.N {
-					fmt.Printf("i = %d, j = %d\n", i, j)
+					if verbose {
```

Comment:
Should there be tests for verbose?

Reply:
i don't think so, it just gives us extra prints to the screen for debugging.